Oscil.h

```diff
@@ -203,6 +217,19 @@ class Oscil
 		}
 	}
 
+  	/** Set the frequency using UFixMath<24,8> fixed-point number format.
+	This might be faster than the float version for setting low frequencies such as
+	1.5 Hz, or other values which may not work well with your table size. A UFixMath<24,8>
+	representation of 1.5 is 384 (ie. 1.5 * 256). Can't be used with UPDATE_RATE
+	less than 64 Hz.
+	@param frequency in UFixMath<24,8> fixed-point number format.
+	*/
+        inline
+	void setFreq(UFixMath<24,8> frequency)
```

Comment:
~~See above. A template function would be much more powerful.~~

Nevermind. However, I suggest moving the template-version of setFreq(), upwards, and declaring this one as a template-specialization, explicitly.

Reply:
Done (I think?)